### ISSUE TITLE
Error starting dotcms in WebSphere 

### ISSUE THREAD

**Issue by bryanboza** [dotCMS/core#7794]
We are having an error with the cron expression, when you are starting a dotcms instance running WebSphere v8.5.5 
```
[28/05/15 15:13:46:931 CST] ERROR init.DotInitScheduler: Day-of-Week values must be between 1 and 7
java.text.ParseException: Day-of-Week values must be between 1 and 7
	at org.quartz.CronExpression.addToSet(CronExpression.java:920)
	at org.quartz.CronExpression.storeExpressionVals(CronExpression.java:630)
	at org.quartz.CronExpression.buildExpression(CronExpression.java:430)
	at org.quartz.CronExpression.<init>(CronExpression.java:255)
	at org.quartz.CronTrigger.setCronExpression(CronTrigger.java:471)
	at org.quartz.CronTrigger.<init>(CronTrigger.java:381)
	at com.dotmarketing.init.DotInitScheduler.start(DotInitScheduler.java:756)
	at com.dotmarketing.servlets.InitServlet.init(InitServlet.java:202)
	at com.ibm.ws.webcontainer.servlet.ServletWrapper.init(ServletWrapper.java:330)
	at com.ibm.ws.webcontainer.servlet.ServletWrapper.loadOnStartupCheck(ServletWrapper.java:1421)
	at com.ibm.ws.webcontainer.webapp.WebApp.doLoadOnStartupActions(WebApp.java:1179)
	at com.ibm.ws.webcontainer.webapp.WebApp.commonInitializationFinally(WebApp.java:1147)
	at com.ibm.ws.webcontainer.webapp.WebApp.initialize(WebApp.java:1053)
	at com.ibm.ws.webcontainer.webapp.WebApp.initialize(WebApp.java:6404)
	at com.ibm.ws.webcontainer.osgi.DynamicVirtualHost.startWebApp(DynamicVirtualHost.java:446)
	at com.ibm.ws.webcontainer.osgi.DynamicVirtualHost.createRunnableHandler(DynamicVirtualHost.java:248)
	at com.ibm.ws.webcontainer.osgi.DynamicVirtualHost.createRunnableHandler(DynamicVirtualHost.java:313)
	at com.ibm.ws.http.internal.VirtualHostImpl.discriminate(VirtualHostImpl.java:241)
	at com.ibm.ws.http.dispatcher.internal.channel.HttpDispatcherLink.ready(HttpDispatcherLink.java:306)
	at com.ibm.ws.http.channel.internal.inbound.HttpInboundLink.handleDiscrimination(HttpInboundLink.java:449)
	at com.ibm.ws.http.channel.internal.inbound.HttpInboundLink.handleNewRequest(HttpInboundLink.java:383)
	at com.ibm.ws.http.channel.internal.inbound.HttpInboundLink.processRequest(HttpInboundLink.java:283)
	at com.ibm.ws.http.channel.internal.inbound.HttpInboundLink.ready(HttpInboundLink.java:254)
	at com.ibm.ws.tcpchannel.internal.NewConnectionInitialReadCallback.sendToDiscriminators(NewConnectionInitialReadCallback.java:174)
	at com.ibm.ws.tcpchannel.internal.NewConnectionInitialReadCallback.complete(NewConnectionInitialReadCallback.java:83)
	at com.ibm.ws.tcpchannel.internal.WorkQueueManager.requestComplete(WorkQueueManager.java:504)
	at com.ibm.ws.tcpchannel.internal.WorkQueueManager.attemptIO(WorkQueueManager.java:574)
	at com.ibm.ws.tcpchannel.internal.WorkQueueManager.workerRun(WorkQueueManager.java:929)
	at com.ibm.ws.tcpchannel.internal.WorkQueueManager$Worker.run(WorkQueueManager.java:1018)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

Tested on the last nightly build // Postgres // WS v8.5.5

**Comment by maurizo** [comment]
Could not reproduce it.

**Comment by jtesser** [comment]
@maurizo did you check with @bryanboza here?

**Comment by jgambarios** [comment]
@maurizo see comment above https://github.com/dotCMS/core/issues/7794#issuecomment-114554879

**Comment by maurizo** [comment]
Yes I did, none of the instances created for testing presented the problem.

**Comment by bryanboza** [comment]
Fixed, tested on WS v8.5 // dotcms_3.3 // Linux
